src/appfl/comm/globus_compute/globus_compute_client_communicator.py

```diff
@@ -103,3 +103,129 @@ def globus_compute_client_entry_point(
             raise NotImplementedError(
                 f"Task {task_name} is not implemented in the client endpoint."
             )
+
+
+def globus_compute_client_entry_point_ds(
+    task_name="N/A",
+    client_agent_config_file=None,
+    model_file=None,
+    meta_data_file=None,
+):
+    """
+    Entry point for the Globus Compute client endpoint for federated learning.
+    :param `task_name`: The name of the task to be executed.
+    :param `client_agent_config`: The configuration for the client agent.
+    :param `model`: [Optional] The model to be used for the task.
+    :param `meta_data`: [Optional] The metadata for the task.
+    :return `model_local`: The local model after the task is executed. [Return `None` if the task does not return a model.]
+    :return `meta_data_local`: The local metadata after the task is executed. [Return `{}` if the task does not return metadata.]
+    """
```

Comment:
Update the description and explain why this function is needed in addition to `globus_compute_client_entry_point`.